src/components/copy-button/_copy-button.scss

```diff
@@ -23,25 +23,30 @@
     top: 1.2rem;
     left: 50%;
 
+    &:focus {
+      border: 2px solid red;
+    }
+
     &:before {
       @include layer('overlay');
       @include typescale('omega');
       top: 1.1rem;
-      padding: 0.5rem 1rem;
-      border: 1px solid $ui-03;
-      color: $text-01;
+      padding: $spacing-2xs;
+      color: $inverse-01;
       content: attr(data-feedback);
       transform: translateX(-50%);
       white-space: nowrap;
       pointer-events: none;
+      border-radius: 4px;
+      font-weight: 400;
     }
 
     &:after {
       top: 0.85rem;
-      width: 0.5rem;
```

Comment:
I think this just needs a `left: -.3px;` to center the top arrow 

<img width="247" alt="screen shot 2018-05-11 at 1 46 29 pm" src="https://user-images.githubusercontent.com/11928039/39942006-d99b7d78-5523-11e8-94ac-18b450117764.png">

<img width="650" alt="screen shot 2018-05-11 at 1 54 52 pm" src="https://user-images.githubusercontent.com/11928039/39942011-de3f18bc-5523-11e8-96e3-6b594491e02c.png">